sdk/keyvault/keyvault-keys/README.md

```diff
@@ -301,154 +421,328 @@ versions of a specific key. The following API methods are available:
-  for (let deletedKey of page) {
-    console.log("Deleted key: ", deletedKey);
+  for await (let page of client.listDeletedKeys().byPage()) {
+    for (let deletedKey of page) {
+      console.log("Deleted key: ", deletedKey);
+    }
   }
-}
-for await (let page of client.listPropertiesOfKeyVersions(keyName).byPage()) {
-  for (let versionProperties of page) {
-    console.log("Version: ", versionProperties);
+  for await (let page of client.listPropertiesOfKeyVersions(keyName).byPage()) {
+    for (let versionProperties of page) {
+      console.log("Version: ", versionProperties);
+    }
   }
 }
 ```
 
 ## Cryptography
 
 This library also offers a set of cryptographic utilities available through
-`CryptographyClient`. Similar to the `KeysClient`, `CryptographyClient` will
+`CryptographyClient`. Similar to the `KeyClient`, `CryptographyClient` will
 connect to Azure Key Vault with the provided set of credentials. Once
 connected, `CryptographyClient` can encrypt, decrypt, sign, verify, wrap keys,
 and unwrap keys.
 
-We can next connect to the key vault service just as we do with the KeysClient.
+We can next connect to the key vault service just as we do with the KeyClient.
 We'll need to copy some settings from the key vault we are
 connecting to into our environment variables. Once they are in our environment,
 we can access them with the following code:
 
 ```typescript
 import { DefaultAzureCredential } from "@azure/identity";
-import { KeysClient, CryptographyClient } from "@azure/keyvault-keys";
+import { KeyClient, CryptographyClient } from "@azure/keyvault-keys";
 
-// DefaultAzureCredential expects the following three environment variables:
-// * AZURE_TENANT_ID: The tenant ID in Azure Active Directory
-// * AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
-// * AZURE_CLIENT_SECRET: The client secret for the registered application
 const credential = new DefaultAzureCredential();
 
-// Build the URL to reach your key vault
 const vaultName = "<YOUR KEYVAULT NAME>";
 const url = `https://${vaultName}.vault.azure.net`;
 
-// Connect to the key vault service
-const keysClient = new KeysClient(url, credential);
+const keysClient = new KeyClient(url, credential);
 
-// Create or retrieve a key from the keyvault
-let myKey = await keysClient.createKey("MyKey", "RSA");
+async function main() {
+  // Create or retrieve a key from the keyvault
+  let myKey = await keysClient.createKey("MyKey", "RSA");
 
-// Lastly, create our cryptography client and connect to the service
-// This example uses the URL that is part of the key we created (called key ID or kid)
-const cryptographyClient = new CryptographyClient(url, myKey.keyMaterial!.kid!, credential);
+  // Lastly, create our cryptography client and connect to the service
+  // This example uses the URL that is part of the key we created (called key ID or kid)
+  const cryptographyClient = new CryptographyClient(url, myKey.keyMaterial!.kid!, credential);
```

Comment:
The constructor signature for CryptographyClient has changed in the master branch as of last night :)

Reply:
Update here and in the samples that follow

Reply:
I pulled from master, made the fix, and pushed to the branch in this PR